README.md

```diff
@@ -26,7 +27,9 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose
+  @person = person
```

Comment:
No need for an instance variable here, those only work when we're inside of a class definition. Instead, we want a parameter (aka argument) for this method, like so:

```ruby
def offer_rose(person)
  puts "would you take this rose and help out an old beggar #{person}"
end
```